cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -427,6 +455,21 @@ func (c *Client) AddMember(name string, peerAddrs string) ([]Member, error) {
 	return ret, nil
 }
 
+func memberPromoteWithRety(ctx context.Context, cli *clientv3.Client, learnerID uint64) error {
+	klog.V(1).Infof("[etcd] Promoting a learner as a voting member: %016x", learnerID)
+	err := wait.PollImmediate(15*time.Second, 10*time.Minute, func() (bool, error) {
+		// TODO: warning logs from etcd client should be removed
```

Comment:
can you plesse explain what these warnings are and can we solve the todo with this pr?

Reply:
+1, if this is TODO item in a following pr, then it could be helpful for someone who wants to fix it with more context.

Reply:
Add some comments here.